### ISSUE TITLE
Cannot decrypt or build CIA from any game cartridge

### ISSUE THREAD

**Issue by xSlake** [d0k3/GodMode9#384]
As the title says I cant seem to decrypt any games via. GM9 and i am wondering why.
It just says the Tilte of the game and then "Dexryption failed".
I have tired to decrypt multiple cartridges, but none seems to work.
Ive looked through older posts and found one but the issue hasnt been solved
there either.
I would appreciate it if someone could help me.
Tyvm 
xSlake

**Comment by xSlake** [comment]
Could the problem be my 2 gig SD card? :/

**Comment by urherenow** [comment]
Yes. First: you need enough space, so get a 32gb or better. Second, you need to make sure you have a PROPER aeskeydb.bin in the proper location. Third, guthub it not a support forum. Please ask your questions on discord or gbatemp. Post here only when you’ve discovered a bug.

**Comment by MadScript77** [comment]
Considering 3DS carts go up to 4 GB, I would consider an 8 GB card to be the bare minimum for dumping games.  That's with NOTHING installed, mind you.  If you actually want to install games, you should go with a 32 GB at the very least (I'd recommend a 64 GB one, personally).

You can't decrypt something that isn't encrypted in the first place, either.  You can only decrypt a CIA created with the encrypted title key (requires a proper aeskeydb.bin and seeddb.bin in gm9/support).  Meaning the game has to come from the eShop, not a cart dump.  ROM to CIA is a dirty hack.  The information to create a semi-genuine encrypted CIA just isn't present on the cart.  There is nothing to decrypt, because there was nothing to encrypt with in the first place.

With that, I'll second that "this isn't a support forum" and motion to close this.  Please ask for help on the GBATemp forum, the official Discord server, or the IRC support channel on Freenode before opening an issue on Github.
